package.json

```diff
@@ -4,60 +4,61 @@
   "license": "MIT",
   "scripts": {
     "ng": "ng",
-    "build": "ng build",
+    "build": "ng build --prod",
     "test": "ng test",
     "lint": "ng lint",
-    "e2e": "ng e2e"
+    "e2e": "ng e2e",
+    "start": "cd stripe-node && npm run start"
```

Comment:
Would `npm run start` be calling itself?

Reply:
Because it is in a subdir so it is running npm start there